app/update_checkers/node.rb

```diff
@@ -7,13 +7,19 @@ class Node < Base
     def latest_version
       @latest_version ||=
         begin
-          url = URI("http://registry.npmjs.org/#{dependency.name}")
-          JSON.parse(Net::HTTP.get(url))["dist-tags"]["latest"]
+          JSON.parse(Net::HTTP.get(dependency_url))["dist-tags"]["latest"]
         end
     end
 
     def dependency_version
       Gem::Version.new(dependency.version)
     end
+
+    private
+
+    def dependency_url
+      path = dependency.name.gsub("/", "%2F")
```

Comment:
That'll crush the `@` too I'm afraid, and it needs to be in tact here. From my understanding the `/` should be the only thing that needs to be encoded, so it seems appropriate to explicitly handle just that (unless I'm totally wrong and there are other characters to handle...)